projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/b2b-checkout.ts

```diff
@@ -475,17 +542,14 @@ export function interceptPaymentTypesEndpoint(): string {
   return alias;
 }
 
-export function interceptCheckoutB2BDetailsEndpoint(fixture: string) {
+export function interceptCheckoutB2BDetailsEndpoint() {
   const alias = 'getCheckoutDetails';
-  cy.intercept(
-    {
-      method: 'GET',
-      path: `${Cypress.env('OCC_PREFIX')}/${Cypress.env(
-        'BASE_SITE'
-      )}/users/**/carts/**/*?fields=deliveryAddress(FULL),deliveryMode(FULL),paymentInfo(FULL),costCenter(FULL),purchaseOrderNumber,paymentType(FULL)*`,
-    },
-    { fixture, statusCode: 200 }
```

Comment:
Removed stub, reason:
Address ID mismatch between mock Address obj (in checkout-details.json) and server Address entity (PK value) makes test fails.
Using the server response rather than mock object to avoid this situation.